.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+<!--
+    Please sign CNCF's Developer Certificate of Origin and sign-off your commits by adding the -s / --sign-off flag to `git commit`
+
+    More information on both of those can be found at https://github.com/apps/dco
```

Comment:
Don't you think that users would be confused that they need to configure that app?

Reply:
I think it's clear enough, but we can always add a new URL if we find a better one.